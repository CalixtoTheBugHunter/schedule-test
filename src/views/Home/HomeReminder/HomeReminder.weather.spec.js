/* eslint-disable no-undef */
import { mount } from '@vue/test-utils'
import Weather from './HomeReminder.weather'

describe('Weather', () => {
  const WRAPPER = mount(Weather, {
    props: {
        value: {}
    },
  })

  it('Component exists', () => {
    expect(WRAPPER.exists()).toBe(true)
  })

  it('Component not changed', () => {
    expect(WRAPPER.html()).toMatchSnapshot()
  })
})