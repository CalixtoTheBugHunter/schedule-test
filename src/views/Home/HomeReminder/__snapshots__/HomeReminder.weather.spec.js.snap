// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weather Component not changed 1`] = `<p class="ReminderWeather"><img src="http://openweathermap.org/img/w/undefined.png"> </p>`;
